client/locales/de.json

```diff
@@ -0,0 +1,402 @@
+        "accessControl": {
+            "pageTitle": "Zugangskontrolleinstellungen",
+            "httpAuth": "HTTP Authentifizierungs Einstellungen",
+            "username": "Benutzername",
+            "password": "Passwort",
+            "passwordRepeat": "Passwort (wiederholen)",
+            "saveCredentials": "Eingaben speichern",
+            "sshKeys": "SSH Schlüssel Einstellungen",
+            "sshRSA": "ssh-rsa ...",
+            "saveKeys": "Speichere SSH Schlüssel",
+            "confirmDisableKeys": "\"confirm\" eingeben. Schliess Dich nicht aus!",
+            "disableKeys": "Dauerhaft das hochladen von SSH Schlüssel abschalten",
+            "passwordsMismatch": "Passwörter stimmen nicht überein."
+        },
+        "carpetMode": {
+            "pageTitle": "Teppichmodus Einstellungen",
+            "explanation": "Im Teppichmodus erkennt der Saugroboter Teppiche automatisch und erhöht entsprechend die Saugleistung.",
+            "warning": "Ändern Sie einen der folgenden Parameter nur, wenn Sie wissen, was Sie tun!",
+            "currentLow": "Momentan Niedrig",
+            "currentHigh": "Momentan Hoch",
+            "currentIntegral": "Momentan Integral",
+            "stallTime": "Standzeit",
+            "confirm": "Sollen die im Teppichmodus vorgenommenen Änderungen wirklich gespeichert werden?"
+        },
+        "cleaningHistory": {
+            "header": "Letzte Reinigungsläufe",
+            "loadMore": "Mehr...",
+            "showMap": "Reinigungskarte anzeigen",
+            "historyRecord": "#{{id}} gestartet am {{startDate}}",
+            "duration": "Dauer",
+            "area": "Bereich",
+            "completed": "Vollständig",
+            "code": "Code",
+            "unknownState": "clean_record_map hat unbekannten Zustand zurückgegeben: {{state}}",
+            "mapTitle": "Reinigungskarte von {{cleaningDate}}"
+        },
+        "consumables": {
+            "mainBrush": "Hauptbürste",
+            "sideBrush": "Seitenbürste",
+            "filter": "Filter",
+            "sensorCleaning": "Sensorreinigung",
+            "allStatistics": "Gesamtstatistik",
+            "totalCleanedArea": "Gesamter gereinigter Bereich:",
+            "totalHours": "Gesamte Reinigungsstunden:",
+            "totalCount": "Gesamte Reinigungsanzahl:",
+            "hoursLeft": "{{hours}} Stunde verbleibend",
+            "hoursLeft_plural": "{{hours}} Stunden verbleibend",
+            "confirm": "Soll dieses Verbrauchsmaterial zurückgesetzt werden?"
+        },
+        "info": {
+            "system": "System",
+            "firmwareVersion": "Firmware Version:",
+            "firmwareBuild": "Firmware Build:",
+            "firmwareUpdate": "Firmware Update:",
+            "firmwareUpdateLink": "[send request]",
+            "currentToken": "Aktueller Token",
+            "token": "Token:",
+            "appLocale": "App-Gebietsschema",
+            "appLocaleName": "Name:",
+            "appLocaleBom": "Bom:",
```

Comment:
What is this?

Reply:
I only know it as the meaning "bill of materials" but this would be incorrect in a firmware wouldn't it?

Reply:
I just searched a little - all the "appLocaleXYZ" variables are no standard abbreviations but something internally defined by Roborock that seem to be related to which servers are used, how the robot is handled by the server and which language file it uses by default - chinese or english.

So basically the "app locale" is not named correctly -"robot locale" and "Roboter-Gebietsvariablen" would be more fitting, since both the valuetudo robot language and GUI language is set somewhere else, independently from these variables.
With this, I would rename the appLocale to something that makes it crystal clear that this has nothing to do with valetudo - and leave the subvariables exactly as they are in roborock.conf where they come from. Maybe like this:
            "appLocale": "Roborock-Systemgebietsvariablen",
            "appLocaleName": "\\"name\\":",
            "appLocaleBom": "\\"bom\\":",
            "appLocaleLocation": "\\"location\\":",
            "appLocaleLanguage": "\\"language\\":",
            "appLocaleTimezone": "\\"timezone\\":",
            "appLocaleLogserver": "\\"logserver\\":",

Reply:
Sounds logical. I changed it to "Roborock-Systemgebietsvariablen". I think we can keep the translations for the variables. But still have no clue what the Bom is about.

Reply:
Me neither - I only know that it changes depending on wheter the robot should load chinese or english language files from Roborock. The logic behind this name remains the secret of the Roborock engineers :-)

I abhore "Denglisch", but this is kind of an exception: the user cannot edit these variables in Valetudo, he has to fire up vi on the robot via SSH and directly edit the file - _where the variables are in english_. In such a case I would be bothered if the variable names are translated in the GUI for no obvious reason instead of treating them as names, forcing me to figure out which one might be which in the file I need to edit... This (and the hope that users who have no clue what a system variable might be have a higher chance of simply ignoring the english words as they should) is why I would prefer to keep the names as they are in the config file and add the quotation marks around them.